packages/database/src/migrations/20210603064215-migrateFijiFacilitiesToNewSubDivisions-modifies-data.js

```diff
@@ -0,0 +1,251 @@
+  FJ_119: 'FJ_sdMacuata',
+  FJ_190: 'FJ_sdLakeba',
+  FJ_046: 'FJ_sdNaitasiri',
+  FJ_052: 'FJ_sdLautoka_Yasawa',
+  FJ_133: 'FJ_sdCakaudrove',
+  FJ_021: 'FJ_sdNausori',
+  FJ_071: 'FJ_sdNadi',
+  FJ_212: 'FJ_sdNausori',
+  FJ_140: 'FJ_sdCakaudrove',
+  FJ_153: 'FJ_sdBua',
+  FJ_012: 'FJ_sdSerua_Namosi',
+  FJ_011: 'FJ_sdSerua_Namosi',
+  FJ_018: 'FJ_sdSerua_Namosi',
+  FJ_072: 'FJ_sdNadi',
+  FJ_164: 'FJ_sdLomaiviti',
+  FJ_137: 'FJ_sdCakaudrove',
+  FJ_030: 'FJ_sdTailevu',
+  FJ_109: 'FJ_sdRa',
+  FJ_005: 'FJ_sdSuva',
+  FJ_097: 'FJ_sdNavosa',
+  FJ_198: 'FJ_sdLakeba',
+  FJ_187: 'FJ_sdLakeba',
+  FJ_184: 'FJ_sdLakeba',
+  FJ_062: 'FJ_sdLautoka_Yasawa',
+  FJ_145: 'FJ_sdTaveuni',
+  FJ_230: 'FJ_sdLomaiviti',
+  FJ_034: 'FJ_sdTailevu',
+  FJ_134: 'FJ_sdCakaudrove',
+  FJ_090: 'FJ_sdNadroga',
+  FJ_003: 'FJ_sdSuva',
+  FJ_102: 'FJ_sdRa',
+  FJ_100: 'FJ_sdRa',
+  FJ_236: 'FJ_sdRa',
+  FJ_214: 'FJ_sdSerua_Namosi',
+  FJ_172: 'FJ_sdKadavu',
+  FJ_033: 'FJ_sdTailevu',
+  FJ_206: 'FJ_sdLomaloma',
+  FJ_205: 'FJ_sdLomaloma',
+  FJ_229: 'FJ_sdSuva',
+  FJ_053: 'FJ_sdLautoka_Yasawa',
+  FJ_004: 'FJ_sdSuva',
+  FJ_132: 'FJ_sdCakaudrove',
+  FJ_130: 'FJ_sdCakaudrove',
+  FJ_129: 'FJ_sdCakaudrove',
+  FJ_118: 'FJ_sdMacuata',
+  FJ_086: 'FJ_sdNadroga',
+  FJ_085: 'FJ_sdNadroga',
+  FJ_060: 'FJ_sdLautoka_Yasawa',
+  FJ_173: 'FJ_sdKadavu',
+  FJ_226: 'FJ_sdSuva',
+  FJ_002: 'FJ_sdSuva',
+  FJ_009: 'FJ_sdSuva',
+  FJ_178: 'FJ_sdKadavu',
+  FJ_099: 'FJ_sdNadroga',
+  FJ_081: 'FJ_sdTavua',
+  FJ_235: 'FJ_sdTavua',
+  FJ_080: 'FJ_sdTavua',
+  FJ_127: 'FJ_sdCakaudrove',
+  FJ_055: 'FJ_sdLautoka_Yasawa',
+  FJ_107: 'FJ_sdRa',
+  FJ_035: 'FJ_sdTailevu',
+  FJ_192: 'FJ_sdLakeba',
+  FJ_131: 'FJ_sdCakaudrove',
+  FJ_204: 'FJ_sdLomaloma',
+  FJ_098: 'FJ_sdNavosa',
+  FJ_123: 'FJ_sdMacuata',
+  FJ_176: 'FJ_sdKadavu',
+  FJ_006: 'FJ_sdSuva',
+  FJ_185: 'FJ_sdLakeba',
+  FJ_199: 'FJ_sdLakeba',
+  FJ_093: 'FJ_sdNadroga',
+  FJ_220: 'FJ_sdTailevu',
+  FJ_091: 'FJ_sdNadroga',
+  FJ_031: 'FJ_sdTailevu',
+  FJ_121: 'FJ_sdMacuata',
+  FJ_058: 'FJ_sdLautoka_Yasawa',
+  FJ_144: 'FJ_sdTaveuni',
+  FJ_146: 'FJ_sdTaveuni',
+  FJ_037: 'FJ_sdNaitasiri',
+  FJ_036: 'FJ_sdNaitasiri',
+  FJ_169: 'FJ_sdKadavu',
+  FJ_168: 'FJ_sdKadavu',
+  FJ_108: 'FJ_sdRa',
+  FJ_122: 'FJ_sdMacuata',
+  FJ_042: 'FJ_sdNaitasiri',
+  FJ_111: 'FJ_sdRa',
+  FJ_222: 'FJ_sdNausori',
+  FJ_020: 'FJ_sdNausori',
+  FJ_114: 'FJ_sdMacuata',
+  FJ_217: 'FJ_sdBua',
+  FJ_017: 'FJ_sdSerua_Namosi',
+  FJ_143: 'FJ_sdTaveuni',
+  FJ_232: 'FJ_sdTaveuni',
+  FJ_095: 'FJ_sdNavosa',
+  FJ_148: 'FJ_sdTaveuni',
+  FJ_152: 'FJ_sdBua',
+  FJ_056: 'FJ_sdLautoka_Yasawa',
+  FJ_057: 'FJ_sdLautoka_Yasawa',
+  FJ_243: 'FJ_sdLautoka_Yasawa',
+  FJ_063: 'FJ_sdLautoka_Yasawa',
+};
+
+exports.up = async function (db) {
+  Object.keys(facilitiesToMigrate).map(async facility => {
+    const parentId = (
+      await db.runSql(`select id from "entity" where code = '${facilitiesToMigrate[facility]}';`)
+    ).rows[0].id;
+    db.runSql(`
```

Comment:
I have a feeling that it will still work, but normally would need `await` here. And perhaps the above should be `.forEach` instead

Reply:
There is an await? For the parentId do you mean?

Reply:
for `db.runSql()` I mean